.github/workflows/diff.yaml

```diff
@@ -262,6 +262,15 @@ jobs:
           cd build
           ctest -R memgraph__unit --output-on-failure -j$THREADS
 
+      - name: Run simulation tests
```

Comment:
Could you please also add the simulation tests to the code analysis build? It runs the tests using the sanitizers, thus I think it might point out memory issues using the simulation tests.